Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Keep hub.Dockerfile aligned to this file as far as possible
 
-ARG base=hmctspublic.azurecr.io/base/node:12-stretch-slim
+ARG base=hmctspublic.azurecr.io/base/node:12-alpine
```

Comment:
@nathan-clark can you check with PE if there are any issues with using alpine?

Reply:
fyi : https://github.com/search?q=org%3Ahmcts+12-alpine&type=Code